types/office-js-preview/index.d.ts

```diff
@@ -21665,29 +21669,27 @@ declare namespace Excel {
          * Fills range from the current range to the destination range using the specified AutoFill logic. 
          * The destination range can be null, or can extend the source either horizontally or vertically. 
          * If the destination range is null, data is filled out based on the surrounding cells (which is the behavior when double-clicking the UI’s range fill handle). 
-         * Discontiguous ranges are not supported.
          * 
          * For more information, read {@link https://support.office.com/article/video-use-autofill-and-flash-fill-2e79a709-c814-4b27-8bc2-c4dc84d49464 | Use AutoFill and Flash Fill}.
          *
-         * [Api set: ExcelApi 1.9]
+         * [Api set: ExcelApi 1.9, ExcelApi Preview for null `destinationRange`]
          *
-         * @param destinationRange The destination range to autofill.
-         * @param autoFillType The type of autofill. Specifies how the destination range is to be filled, based on the contents of the current range. Default is "FillDefault".
+         * @param destinationRange The destination range to AutoFill. Discontiguous ranges are not supported.
```

Comment:
In this case, seems like autofill should be the verb, not the feature.

```suggestion
         * @param destinationRange The destination range to autofill. Discontiguous ranges are not supported.
```